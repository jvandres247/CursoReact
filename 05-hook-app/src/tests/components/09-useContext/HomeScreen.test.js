import React, { useReducer } from "react";
import "@testing-library/jest-dom";
import { mount, shallow } from "enzyme";
import { HomeScreen } from "../../../components/09-useContext/HomeScreen";
import { UserContext } from "../../../components/09-useContext/UserContext";

describe("Pruebas en <HomeScree/>", () => {
  const user = {
    name: "Andres",
    email: "jvandres247@gmail.com",
  };

  const wrapper = mount(
    <UserContext.Provider value={{ user }}>
      <HomeScreen />
    </UserContext.Provider>
  );
  test("Debe de mostrarse correctamente", () => {
      expect(wrapper).toMatchSnapshot();
  });
});
