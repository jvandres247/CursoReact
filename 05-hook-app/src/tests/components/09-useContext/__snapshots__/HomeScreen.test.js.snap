// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScree/> Debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Andres",
   "email": "jvandres247@gmail.com"
}
  </pre>
</div>
`;
